hips/utils/healpix.py

```diff
@@ -0,0 +1,46 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""HEALpy wrapper functions.
+
+This module contains wrapper functions around HEALPix utilizing
+the healpy library
+"""
+
+__all__ = [
+    'boundaries',
+]
+
+import healpy
+import numpy as np
+
+
+def boundaries(nside: int, pix: int, nest: bool=False) -> tuple:
+    """Returns an array containing the angle (theta and phi) in radians.
+
+    This method calls `healpy.boundaries` and `healpy.vec2ang`.
```

Comment:
method -> function

Usually the word method is used for a function in a class, such as this would be the `ham` method:
```
class Spam:
    def ham(self):
        print('spam ham')
```
Standalone functions are always called functions, not methods.